README.md

```diff
@@ -17,3 +17,12 @@
 * Create a new folder under `src/pages/` with the same name of the new xml file.
 * Create `index.jsx` and bootstrap the page using `@splunk/react-page`.
 * Restart Splunk, your new page should shows up.
+
+
+# Package the app
+
```

Comment:
Can you add one line here that tells people docker is required for performing these actions?